### PR TITLE
Review

### PR DESCRIPTION
**Pull request by orchidenver**
Changed: 
1. Changed condiniton of showing loading spinner. Instead of returning component depending on loading state approach was changed to returning part of child components depending on loading state.
2. Added readme file. File is needed to be filled in future.

### CODE REVIEW

File: src/pages/Dashboard.js [hunk 1]
```diff
@@ -12,28 +12,17 @@ import { GithubContext } from '../context/context';
...
-        </Helmet>
-        <main>
-          <Navbar />
-          <Search />
-          <img className='loading-img' src={loader} alt="loading information" />
-        </main>
-      </>
-    )
-  }
+  const ifLoadingContent = loading ? (
```
Comment (ivan-ivan) [inline src/pages/Dashboard.js]: It's better to give such names to boolean variable as `isLoading` `isActive` etc. 

Here's a good article on best practices for boolean variable naming

File: src/pages/Dashboard.js [hunk 2]
```diff
@@ -51,9 +40,7 @@ const Dashboard = () => {
       <main>
         <Navbar></Navbar>
         <Search />
-        <Info />
-        <User />
-        <Repos />
+        {ifLoadingContent}
```
Comment (ivan-ivan) [inline src/pages/Dashboard.js]: You can add this logic directly in the markup here without saving it to the `ifLoadingContent` variable

Or if you decide to leave it as is, it's better to rename the variable to something more descriptive

**Review by ivan-ivan — CHANGES_REQUESTED** [review]
Generally looks good. Just left a few comments

File: src/pages/Dashboard.js (hunk 1 above)
Comment (orchidenver) [reply src/pages/Dashboard.js]: Thanks. I decided to delete a variable and place the whole logic of rendering child components into component itself.

**Review by ivan-shytikov — APPROVED** [review]
Looks good to me

**Review by ivan-ivan — APPROVED** [review]

**Pull request merged**